tests/rptest/tests/partition_balancer_test.py

```diff
@@ -31,6 +31,15 @@
 # consumer timeout than the default 30 seconds
 CONSUMER_TIMEOUT = 90
 
+# TODO: remove after https://github.com/redpanda-data/redpanda/issues/5154
+# is fixed
+LOG_ALLOW_LIST = CHAOS_LOG_ALLOW_LIST + [
+    # ERROR 2022-08-05 06:32:44,034 [shard 0]
+    # rpc - Service handler threw an exception:
+    # raft::offset_monitor::wait_aborted (offset monitor wait aborted)
+    "rpc - .* raft::offset_monitor::wait_aborted"
```

Comment:
curious if this is an actual error, or should instead be dropped on the floor (like we do with a lot of gate_closed, etc...) or logged at a lower level?

Reply:
I've looked at it briefly - mostly it should be ignored I think, but some things that I didn't like:
1. Why use a custom exception class when `ss::abort_requested_exception` exists? Custom exception makes it awkward to e.g. match the exception class at the rpc layer without introducing the dependency on raft subsystem.
2. Looks like this exception is thrown both in case of abort and in case of timeout. I think these should be distinct (timeout shouldn't be ignored as easily as an abort).

So maybe we should just rewrite it to use standard seastar exceptions.

Reply:
> Why use a custom exception class when ss::abort_requested_exception exists?

yeh i wrote the initial version several years ago now and i recall a lot of custom exception use and also general ignorance about all the stuff present in seastar::*.

> Looks like this exception is thrown both in case of abort and in case of timeout. I think these should be distinct

agree about this. in any case, sounds good and looks like a good area for some future cleanup.